shell/platform/android/io/flutter/view/FlutterNativeView.java

```diff
@@ -174,25 +182,25 @@ public void reply(ByteBuffer reply) {
                             throw new IllegalStateException("Reply already submitted");
                         }
                         if (reply == null) {
-                            nativeInvokePlatformMessageEmptyResponseCallback(
+                            mFlutterJNI.nativeInvokePlatformMessageEmptyResponseCallback(
                                     mNativePlatformView, replyId);
                         } else {
-                            nativeInvokePlatformMessageResponseCallback(
+                            mFlutterJNI.nativeInvokePlatformMessageResponseCallback(
                                     mNativePlatformView, replyId, reply, reply.position());
                         }
                     }
                 });
             } catch (Exception ex) {
                 Log.e(TAG, "Uncaught exception in binary message listener", ex);
-                nativeInvokePlatformMessageEmptyResponseCallback(mNativePlatformView, replyId);
+                mFlutterJNI.nativeInvokePlatformMessageEmptyResponseCallback(mNativePlatformView, replyId);
             }
             return;
         }
-        nativeInvokePlatformMessageEmptyResponseCallback(mNativePlatformView, replyId);
+        mFlutterJNI.nativeInvokePlatformMessageEmptyResponseCallback(mNativePlatformView, replyId);
     }
 
     // Called by native to respond to a platform message that we sent.
-    private void handlePlatformMessageResponse(int replyId, byte[] reply) {
+    public void handlePlatformMessageResponse(int replyId, byte[] reply) {
```

Comment:
Can we implement the interfaces as anonymous inner classes?  I'd like to avoid expanding the public API at all so that we have more room with regard to "backwards compatibility" moving forward...